### PR TITLE
Style openstreet

### PR DESCRIPTION
**Pull request by Asam237**
Style the default openstreet map so that it matches our website style

### CODE REVIEW

File: frontend/src/pages/streetMap/StreetMap.page.tsx [hunk 1]
```diff
@@ -46,31 +24,47 @@ const SetViewOnClick = (animateRef: any) => {
...
+  iconUrl: bolt,
+  shadowUrl: bolt,
+
+  iconSize: [0, 0], // size of the icon
+  shadowSize: [30, 30], // size of the shadow
+  shadowAnchor: [10, 45], // the same for the shadow
+});
+
 export const StreetMap = () => {
+  const yaounde: any = [3.826985, 11.495974];
```
Comment (elhmn) [inline frontend/src/pages/streetMap/StreetMap.page.tsx:37]: maybe we can get this value from LATLONG directly to make it more consistent

File: frontend/src/pages/streetMap/StreetMap.page.tsx [hunk 2]
```diff
@@ -46,31 +24,47 @@ const SetViewOnClick = (animateRef: any) => {
...
+            // sticky={false}
+            permanent
+          >
+            <span>{item.name}</span>
+          </Tooltip>
+        </Marker>
+      ))}
       <LocationMarker />
-      <ZoomControl position="bottomright" zoomInText="🧐" zoomOutText="🗺️" />
+      <ZoomControl position="topright" zoomInText="🧐" zoomOutText="🗺️" />
```
Comment (elhmn) [inline frontend/src/pages/streetMap/StreetMap.page.tsx:66]: could we remove the zoomIn and zoomOut Texts ?

**Review by elhmn — APPROVED** [review]
Great work 👍

File: frontend/src/pages/streetMap/StreetMap.page.tsx (hunk 2 above)
Comment (Asam237) [reply frontend/src/pages/streetMap/StreetMap.page.tsx:66]: Oui, nous pouvons supprimer les textes.

File: frontend/src/pages/streetMap/StreetMap.page.tsx (hunk 1 above)
Comment (Asam237) [reply frontend/src/pages/streetMap/StreetMap.page.tsx:37]: Oui, nous pouvons obtenir ces valeurs de LATLONG

**Comment by elhmn** [comment]
Let's merge this 🚀

**Pull request merged**